std/internal/math/biguintcore.d

```diff
@@ -344,6 +344,17 @@ public:
         return buff[z-frontExtraBytes..$];
     }
 
+    /**
+     * Convert to an octal string.
+     */
+    char[] toOctalString() const
+    {
+        auto predictLength = 8*BigDigitBits / 3;
```

Comment:
Perhaps you meant to use the actual length here.
Otherwise, the following code gives a range check error in biguintcore.d on line 1588:

```
import std.array, std.bigint, std.range, std.stdio;
void main () {
    auto a = BigInt ("1" ~ ("0".repeat (10000).join ("")));
    writefln ("%o", a);
}
```